proposals/lip-0028.md

```diff
@@ -3,15 +3,15 @@ LIP: 0028
 Title: Define schema and use generic serialization for transactions
 Author: Iker Alustiza <iker@lightcurve.io>
 Discussions-To: https://research.lisk.io/t/define-schema-and-use-generic-serialization-for-transactions/211
-Type: Informational
+Type: Standards Track
 Created: 2020-02-18
 Updated: 2020-07-21
 Requires: 0027
```

Comment:
The new LIP proposing to replace the `type` property by the `moduleID` and `assetID` property should also be added as requirement.

Reply:
LIP 0036 added as a requirement